types/stripe/index.d.ts

```diff
@@ -95,7 +96,8 @@ declare class Stripe {
     disputes: Stripe.resources.Disputes;
     events: Stripe.resources.Events;
     invoices: Stripe.resources.Invoices;
-    invoiceItems: Stripe.resources.InvoiceItems;
+	invoiceItems: Stripe.resources.InvoiceItems;
```

Comment:
It looks like you're using hard tabs here.  That needs to be spaces to fit with the rest of that project.

Reply:
@pushplay resolved. Let me know how it looks now.